slices/tar.yaml

```diff
@@ -0,0 +1,24 @@
+package: tar
+
+slices:
+  bins:
+    essential:
+      - tar_rmt
+      - tar_tar
+    contents:
+      /usr/sbin/tarcat:
+
+  rmt:
+    essential:
+      - libc6_libs
+    contents:
+      /etc/rmt: {symlink: /usr/sbin/rmt-tar}
```

Comment:
:thinking: 

right...
/etc/rmt -> /usr/sbin/rmt
/usr/sbin/rmt -> /etc/alternatives/rmt
/etc/alternatives/rmt -> /usr/sbin/rmt-tar

I guess this is ok then :)